Src/Sarif.PartnerMatcher.Function/Sarif.PatternMatcher.Function.csproj

```diff
@@ -0,0 +1,40 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <Import Project="$(MSBuildThisFileDirectory)..\..\Targets\build.lib.props" />
+  <Import Project="$(MSBuildThisFileDirectory)..\..\Targets\build.product.props" />
+  
+  <PropertyGroup>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
```

Comment:
><TargetFramework>netcoreapp3.1</TargetFramework> [](start = 4, length = 48)

you can remove the entire line and add the following:
<IsPackable>false</IsPackable>